data/pages.yaml

```diff
@@ -132,16 +132,18 @@ maintainer:
     documentation: Look after collection documentation
   build_community:
     intro: Community maintainers play a special role to enable and encourage contributors.
-    communication: Join or start a community discussion
```

Comment:
removing the `communication` field requires a change here: https://github.com/ansible/jinja-docsite/blob/0496c7846d69033bb9c89d31f034d900d9682ee3/templates/maintainer.html#L28